### DISCUSSION TITLE
Allow webp for backgrounds and profile pictures

### DISCUSSION THREAD

**Discussion by dhalucario** [stoatchat/discussions#501] [Feature Suggestions]
When using webp I can use better quality images with a lot less file size to get the same result. It would be cool if you could support webp.

![image](https://user-images.githubusercontent.com/10533763/220215236-8a1946ac-6417-443f-9f25-7f579fd5d6a9.png)

**Comment by insertish** [comment]
Tracked in this issue: https://github.com/revoltchat/autumn/issues/23
